app/helpers/session_helper.rb

```diff
@@ -5,7 +5,7 @@ def show_session(question)
   end
 
   def show_link(step)
-    "<a href='#{new_registration_path(step)}'>Change</a>".html_safe
+    "<a href='#{new_registration_path(step)}' class='link-color'>Change</a>".html_safe
```

Comment:
Did you run this past Ben?

Reply:
Yes, figured out the issue, put it on another ticket.